### PR TITLE
refactor(use-app-selector-usage)

### PR DESCRIPTION
**Pull request by akhlivnoy**

### CODE REVIEW

File: app/screens/Initial/InitialScreen.tsx [hunk 1]
```diff
@@ -15,25 +15,22 @@ export const InitialScreen: React.FC<IInitialScreenProps> = ({
...
-    state => state.app.loading.isSaveAllQuests,
-  );
-  const isSaveTranslationsLoading = useAppSelector(
-    state => state.app.loading.isSaveTranslations,
+  const { isSaveAllQuests, isSaveTranslations } = useAppSelector(
+    state => state.app.loading,
   );
-
+  //dispatch(questSlice.actions.updateCurrentDay(13));
+  //dispatch(questSlice.actions.setCurrentDayQuestsStack());
```
Comment (Reykjavik151) [inline app/screens/Initial/InitialScreen.tsx]: Why did you add it?

**Review by Reykjavik151 — CHANGES_REQUESTED** [review]
Fix & merge

File: app/screens/Initial/InitialScreen.tsx (hunk 1 above)
Comment (akhlivnoy) [reply app/screens/Initial/InitialScreen.tsx]: I tested day 13 and forgot to delete
